cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -63,6 +63,8 @@ func NewCloudControllerManagerCommand() *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use: "cloud-controller-manager",
+		Deprecated: "please use the cloud controller manager specific " +
```

Comment:
Because of the way external providers vendor `cmd/cloud-controller-manager`, I think this deprecation message would end up in the external providers too?

Reply:
Good point. moved it! thanks @andrewsykim